Update F2 to v1.4.1 with git auto-update config

qrider opened the pull request:
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/OpenF2/F2
 * Official website (optional, not the repository): http://www.openf2.org/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 111
   - Count of watchers: 47
   - Count of forks: 51

# Essential checklist
 * [x] I'm the author of this library - I am current maintainer 
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

PeterDaveHello commented:
Thanks @qrider, please ping me once you pass the build or you need my help.

qrider commented:
Hi @PeterDaveHello, The build passed. Thanks for reaching out. Is there anything else that I need to do to get the new version of the lib on cdnjs?

PeterDaveHello commented:
@qrider Nope, but I got to go to take a rest now, please let me review this on tomorrow morning, thanks.

qrider commented:
Awesome! Thank you!

PeterDaveHello commented:
Hi @qrider,

I found that the files came from different path, however, you didn't set the basePath, and currently, we didn't support multiple filemap on git auto-update, which means, you have to make the file/directory structure the same as they are in your repo, currently I think you can at least set the basePath to `sdk`, or maybe you can consider to add a `dist` folder for the distribution files, so that we can just grab files from `dist` folder, what do you think?

qrider commented:
Let me see if I can add a dist folder.

qrider commented:
Hi @PeterDaveHello, I have added a dist folder to the lib and made the appropriate update in the package.json. basePath is 'dist' and files '*'. Thanks again for the help.

PeterDaveHello commented on ajax/libs/F2/package.json:
```diff
@@ -12,5 +12,13 @@
+    "target": "git://github.com/OpenF2/F2.git",
+    "basePath": "dist",
+    "files": [
+      "*"
```
It'll be even better to use `"**/*"` here for recursive match.

PeterDaveHello commented:
Thanks @qrider. Two problems here:

1. I see `f2.min.js.map` in this commit but didn't see it in your repo.
2. There is no git tag v1.4.1 for the release, though this time you manually submit this RR, new versions update will need the tag for recognition.

qrider commented:
Hi @PeterDaveHello, 
- I have updated the dist folder in the repo to match the 1.4.1 folder in the cdnjs fork. 
- Also tagged the release with 1.4.1.
- Updated the file path in the package.json to "**/*"

Thanks again for the help.

PeterDaveHello commented:
Thanks @qrider, your changes will be deployed within 30 minutes, let's see if it works.

qrider commented:
thanks. verified. looks good.

PeterDaveHello commented:
:+1:
